api/server.go

```diff
@@ -135,7 +135,7 @@ func main() {
 	}
 }
 
-func setupGracefulShutdown(server *http.Server) {
+func setupGracefulShutdown(http_server *http.Server) {
```

Comment:
Since the type is `http.Server`, use `server` as the name is clear enough here. Please keep it short.

Reply:
Since it's a local variable, I prefer a sort name. `svr` or even `s` are good enough.